app/app.py

```diff
@@ -0,0 +1,49 @@
+from flask import Flask, render_template
+from flask_restful import Api
+import logging
```

Comment:
Nitpicks: It would be nice to see module level doc strings. Standard libraries usually go first in alphabetical order.